extra/2-piping.js

```diff
@@ -16,26 +16,31 @@
   the final result to the variable goodCode
 */
 
-function add() {
-
+function add(num1, num2) {
+  return num1 + num2;
 }
 
-function multiply() {
-
+function multiply(value1, value2) {
+  return value1 * value2;
 }
 
-function format() {
-
+function format(number) {
+  return `£${number}`;
 }
 
 const startingValue = 2;
 
 // Why can this code be seen as bad practice? Comment your answer.
-let badCode =
+//readability is sacrificed for concise code. Returning to this at a later date I would need to consult the global functions to figure out each step of the program
+let badCode = format(multiply(add(startingValue, 10), 2));
 
 /* BETTER PRACTICE */
-
-let goodCode =
```

Comment:
Ignore this exercise - there is little guidance here and both are bad practice, as you aren't making the code more readable or succinct

Answer below
```js
// don't use somewhat useless functions, when operators do a better job (and are more readable)
const startingValue = 2
const bestCode = `${(10 + startingValue) * 2}`
```